### ISSUE TITLE
Passing --silent via CLI does not suppress output

### ISSUE THREAD

**Issue by stevenmusumeche** [dotansimha/graphql-code-generator#1683]
First of all, thank you for the work on this project. It is really pushing the boundaries of GraphQL!

I am using the CLI to generate TypeScript, and it works great.

`gql-gen --config ./config/gql-gen/config.yml`

I would now like to also run it, in CI, with the --silent option, like this:

`gql-gen --config ./config/gql-gen/config.yml --silent=true`

I also tried

`gql-gen --config ./config/gql-gen/config.yml --silent`

In both cases, there is still output.  If I add the silent option to my config file, it works.

**Comment by kamilkisiela** [comment]
#1717 should fix it

**Comment by iamdanthedev** [comment]
Btw, is it possible to somehow suppress the type of output as on the screenshot but leave the errors in the output?

```
![image](https://user-images.githubusercontent.com/24467138/56258366-9b35c580-60cf-11e9-9be2-796f35ae47d7.png)
```

**Comment by dotansimha** [comment]
@iamdanthedev can you please provide the exact command and config file you are using? Also, which version is that?

**Comment by dotansimha** [comment]
Fixed in `1.1.0` 🚀
